### PR TITLE
elevate token with webauthn

### PR DESCRIPTION
**Pull request by neurosnap**

### CODE REVIEW

**Comment by madhuravius** [comment]
minor request - I think part of this was that we probably should also highlight the difference between 2fa vs webauthn (why joe was confused in the ticket)

I think we should link to [the docs](https://www.aptible.com/docs/password-authentication#2-factor-authentication-2fa) right next to the 2fa label letting them know to look here. (though the doc itself is pretty light on outlining the difference between software-based 2fa and hardware-based 2fa)

File: src/ui/pages/elevate.tsx [hunk 1]
```diff
@@ -27,21 +27,27 @@ export const ElevatePage = () => {
...
     if (isOtpError) {
       setRequireOtp(true);
+      dispatch(
+        elevateWebauthn({
+          ...loginPayload,
+          webauthn: loader.meta.exception_context.u2f?.payload,
+        }),
+      );
     }
   }, [isOtpError]);
```
[the remark this answers is not in the document]
Comment (madhuravius) [reply src/ui/pages/elevate.tsx:53]: i think in Joe's screenshot he was trying to supply a hardware 2fa to a software 2fa

**Comment by madhuravius** [comment]
minor comment request, lgtm otherwise

File: src/ui/pages/elevate.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (neurosnap) [reply src/ui/pages/elevate.tsx:53]: Screenshot 2023-05-16 at 10 17 59 AM

**Review by madhuravius — APPROVED** [review]

**Pull request merged**